solutions/usaco-831.mdx

```diff
@@ -168,6 +279,8 @@ int main() {
 	written << winners[2].size() << endl;
 }
 ```
-
 </CPPSection>
+<JavaSection>
+Unfortunately, implementing this solution in java is quite tricky, and is beyond the scope of Bronze
```

Comment:
bruh

Reply:
what? You need custom comparators for Java with sorted set...

Reply:
i mean what's wrong with that

Reply:
It's not in the scope of bronze

Reply:
![image](https://user-images.githubusercontent.com/58674441/147625704-db6e2a1b-51dc-4c81-a4f7-facc0b49e0b9.png)

Reply:
so?

Reply:
ik but it's better to not include that in this solution.

Reply:
> this

this solution? what other solution file could there be?